Misc/NEWS.d/next/Library/2018-05-15-17-06-42.bpo-33516.ZzARe4.rst

```diff
@@ -0,0 +1,2 @@
+Add ``__round__`` to the list of magic methods supported by
```

Comment:
Oh, you didn't mention the module. Please add "unittest.mock: " prefix to this NEWS entry.